plugins/log.js

```diff
@@ -0,0 +1,39 @@
+const fs = require("fs");
+const logFile = fs.createWriteStream("screeps.log", { flags: "a" });
+const errorLogFile = fs.createWriteStream("screeps.errors.log", { flags: "a" });
```

Comment:
Can you get the filenames to use from the config file as well? It seems like an obvious place to me, and perhaps someone wants their logs in a "log" subdirectory, for example.

Reply:
Note here that if both of these happen to be the same filename, the logs would go to the same file. No special code is required for that behavior.

Reply:
I agree on this account as well, adding 1 case to the 3 you mentioned above, if errorlogfile is null but normal isn't then all logs go to the same file (as it is currently effectively).